<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(255, 255, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="create_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 20px;
background:rgb(170, 170, 255);
border-width:2px;
border-radius:10px;
border-color:black;
}
QPushButton:hover{
  background:rgb(206, 213, 255);
  border-radius: 10px;

}</string>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>118</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="Selection">
       <property name="text">
        <string>Selection Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Bubble">
       <property name="text">
        <string>Bubble Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Insertion">
       <property name="text">
        <string>Insertion Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Merge">
       <property name="text">
        <string>Merge Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Quick">
       <property name="text">
        <string>Quick Sort</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="Scatter">
       <property name="text">
        <string>Scatter Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Bar">
       <property name="text">
        <string>Bar Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Stem">
       <property name="text">
        <string>Stem Plot</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 20px;
background:rgb(170, 170, 255);
border-width:2px;
border-radius:10px;
border-color:black;
}
QPushButton:hover{
  background:rgb(206, 213, 255);
  border-radius: 10px;

}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 20px;
background:rgb(170, 170, 255);
border-width:2px;
border-radius:10px;
border-color:black;
}
QPushButton:hover{
  background:rgb(206, 213, 255);
  border-radius: 10px;

}</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 20px;
background:rgb(170, 170, 255);
border-width:2px;
border-radius:10px;
border-color:black;
}
QPushButton:hover{
  background:rgb(206, 213, 255);
  border-radius: 10px;

}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>671</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(135, 206, 235)
</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="size_label">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnBars"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>Custom List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="list"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QRadioButton" name="custom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom Data</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="random">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QSlider" name="speed_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Switch-Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sayi">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Big(O):</string>
    </property>
   </widget>
   <widget class="QLabel" name="bigo">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
